build.sh

```diff
@@ -78,7 +78,7 @@ configure() {
     return 0
   else
     echo "--[ Listing errors in config.log ]--"
-    cat config.log | grep error
+    cat config.log | grep error | tail
```

Comment:
Cannot it be grep error config.log | tail ?

Reply:
I have no problem with that. Feel free to make just any edits You like. I'm really sorry but I'm not really in the git reset commands. Every time I do that and try to amend commit changes it automatically closes the pull request and I have to retry all the clicking on github web.

Reply:
BTW: Speaking of grep erro config.log ... I definitely agree it could be changed, but this is more readable in my opinion...